Faraday_Proxy_Tools/FaradayIO/deviceconfig.py

```diff
@@ -78,10 +83,12 @@ def update_basic(self, config_bitmask, callsign, callsign_id, p3_bitmask, p4_bit
             self.basic_gpio_p3_bitmask = p3_bitmask
             self.basic_gpio_p4_bitmask = p4_bitmask
             self.basic_gpio_p5_bitmask = p5_bitmask
+            return True
         else:
             print "ERROR: Callsign too long!"
```

Comment:
Agreed, can we push adding logging to a new pull request and issue enhancement? I don't want this pull to bloat and it has many needed functional updates for first users.